Week1/homework/index.html

```diff
@@ -1 +1,20 @@
-<!-- replace this with your HTML content -->
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Homework - javascript2- week1</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+
+</head>
+
+<body>
+  <div id="root"> </div>
+
+  <script type="text/javascript" src="app.js" src="/image/"></script>
```

Comment:
Which source code this element is using do you think ? and why ?

Reply:
This element use src="app.js", and i delete src="/image/", but still it's generate my images.